users/init.sls

```diff
@@ -32,6 +32,7 @@ include:
   - users.user_files
 {%- endif %}
 {%- endif %}
+  - users.polkit
```

Comment:
Should we force `users.polkit` or simply document in README?

Reply:
I don't think we should force this on everyone as not every OS supported by this formula includes `polkit` in its default installation.  
Maybe have something like `polkit_by_default` in `map.jinja` and have it overwritten with something in `pillar[users-formula]`?

Reply:
Hmm I'm not familiar with `polkit` so agree @0xf10e this could be pillar (default `False`).  @IMBArator Could you update and I'll be happy to review/approve.  thanks!!

Reply:
Moved to https://github.com/saltstack-formulas/users-formula/issues/200